app/lib/three_scale/timed_value.rb

```diff
@@ -1,4 +1,5 @@
-require 'redis'
+# frozen_string_literal: true
+
 class ThreeScale::TimedValue
```

Comment:
I haven't found any reference to this class in the code. I'd say is dead code.

Reply:
The usage seems to have been removed here: https://github.com/3scale/porta/pull/2288/files#diff-8aa693f2978224422216c485c5a77372dd282ca5599f558c185744473e8a10d7

I think we can remove it in a different PR, and then rebase this PR.